dandiapi/urls.py

```diff
@@ -68,6 +69,11 @@ def to_url(self, value):
 register_converter(DandisetIDConverter, 'dandiset_id')
 urlpatterns = [
     path('api/', include(router.urls)),
+    re_path(
```

Comment:
re 302 -> 200 fails, may be this one should come after the `/download/` below so it would get a chance to match that one first??? but then may be this (and the others?) one needs an anchor `$` at the end of the regex?

Reply:
Oh wow, nice catch! Adding the `$` to the end of the regex prevents that one from matching the download URL as well. And indeed, before I added that the `/download/` endpoint magically transformed into another metadata endpoint.

See 6ce80df.

Reply:
cool. shouldn't all the regexes be anchored$ here to avoid some spurious "working" ones?

Reply:
All the patterns that start with `re_path` probably should yes, although it is a moot point as long as there are no overlaps.